Folge_02/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "setup-react",
```

```suggestion
  "name": "02-react-setup",
```

Auch das würde ich passend zur Folge machen.